.travis.yml

```diff
@@ -21,9 +37,5 @@ notifications:
   webhooks: https://www.travisbuddy.com/
   on_success: never
 
-env:
-  global:
-    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true  # Speeds installation of html-proofer
```

Comment:
This didnt seem to work with build stages.
https://github.com/travis-ci/travis-ci/issues/9734